docs/concepts.rst

```diff
@@ -2,7 +2,7 @@
 Concepts
 ==============
 
-The main building blocks of the library are the `Stream`, `Operator` and the `Atifact` classes.
+The main building blocks of the library are the `Stream`, `Operator` and the `Artifact` classes.
```

Comment:
down here, when detailing Stream, it does not look a class, but an abstract term.

Reply:
This page will go through a bit revision.